service/client/client.js

```diff
@@ -613,6 +616,26 @@ export default class Client {
         );
     };
 
+    getFileInfosForPost = async (teamId, channelId, postId) => {
+        return this.doFetch(
+            `${this.getChannelNeededRoute(teamId, channelId)}/posts/${postId}/get_file_infos`,
+            {method: 'get'}
+        );
+    };
+
+    uploadFile = async (teamId, channelId, clientId, fileFormData, formBoundary) => {
```

Comment:
Does the `formBoundary` need to be passed separately or can we always get it from the provided `fileFormData`?